src/components/ModelView/ChartsHolder.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {
+  MainContentWrapper,
+  MainContentInner,
+  MainContentInnerBody,
+} from './ChartsHolder.style';
+import LocationPageHeader from 'components/LocationPageHeader/LocationPageHeader';
+import NoCountyDetail from './NoCountyDetail';
+import ModelChart from 'components/Charts/ModelChart';
+import { Projections } from 'models/Projections';
+import { Projection } from 'models/Projection';
+
+const ChartsHolder = (props: {
```

Comment:
Why do you prefer to do it this way vs ({a:string,b:string..})

Reply:
I don't think you can do that in typescript-- i'd have to do something like 

```
const Component = ({
  a,
  b,
}: {
  a: string;
  b: string;
}) => {...}
```

or hmmmm i guess let me mess with the generic types

Reply:
I think the FunctionComponent expects props: Type